spec/rubocop/cop/style/if_unless_modifier_spec.rb

```diff
@@ -44,7 +42,7 @@ def f
           def f
             # Comment 1
             if condition
-              puts '#{spaces}'
+              puts '#{spaces(59)}'
```

Comment:
I've changed this to `#{body}` in my PR, since in the other examples and more importantly, the `expect_offense` statement above it's used like that.